src/gateway/types.js

```diff
@@ -147,22 +130,12 @@ export type RenderResult = {
  */
 export type RenderAPI = {
     /**
-     * Callback to request the value of a header in the request.
+     * A dict of headers that've come from the client.
      *
-     * This can be used to determine additional context about the render
-     * operation. For example, depending on your specific setup, they may
-     * contain version information to help determine what the render package
-     * should contain. It is provided as a callback so that the gateway
-     * implementation can track which headers influence a render, which can then
-     * be reported back as a Vary header in the gateway response.
-     */
-    +getHeader: GetHeaderCallback,
-
-    /**
-     * Callback to retrieve a map of the request headers that have been accessed
-     * and their values.
+     * Access of these headers should be tracked and a Vary header should be
```

Comment:
Can you explicitly state that all header access must be tracked and added to Vary, not just those that exist?